helpers/wrapper/contracts/Wrapper.sol

```diff
@@ -123,14 +122,8 @@ contract Wrapper {
       wethContract.withdraw(_makerAmount);
       // Transfer ether to the user.
       msg.sender.transfer(_makerAmount);
-    }
-
-    // Ensure no WETH balance remains.
-    require(wethContract.balanceOf(address(this)) == 0,
-      "WETH_BALANCE_REMAINING");
-
-    // Ensure no ether balance remains.
-    require(address(this).balance == 0,
-      "ETH_BALANCE_REMAINING");
+    } else if ((_makerToken != address(0)) && (_takerToken == address(wethContract))) {
```

Comment:
Might change the conditional here to check `_takerWallet == address(0)` and add a comment to make it clear that in this case the contract would have received tokens after the swap and needs to transfer them out.